mlrun/model_monitoring/writer.py

```diff
@@ -124,15 +124,15 @@ def get_v3io_container(project_name: str) -> str:
     def _get_v3io_client() -> V3IOClient:
         return mlrun.utils.v3io_clients.get_v3io_client(
             endpoint=mlrun.mlconf.v3io_api,
-            access_key=os.getenv(ProjectSecretKeys.ACCESS_KEY),
+            access_key=os.getenv(ProjectSecretKeys.PIPELINES_ACCESS_KEY),
         )
 
     @staticmethod
     def _get_v3io_frames_client(v3io_container: str) -> V3IOFramesClient:
         return mlrun.utils.v3io_clients.get_frames_client(
             address=mlrun.mlconf.v3io_framesd,
             container=v3io_container,
-            token=os.getenv(ProjectSecretKeys.ACCESS_KEY, ""),
+            token=os.getenv(ProjectSecretKeys.PIPELINES_ACCESS_KEY, ""),
```

Comment:
From an active deployment:
```sh
k -n default-tenant exec -it nuclio-test-monitoring-app-flow29-model-monitoring-writer-wrknc -- \
printenv | grep ACCESS
```
results in
```txt
MODEL_MONITORING_ACCESS_KEY=***
V3IO_ACCESS_KEY=***
```
`MODEL_MONITORING_PIPELINES_ACCESS_KEY` is not even set in this writer pod. Is it set in your environment?
What error did you face?

Reply:
Updating here following our conversation today:
1. `ProjectSecretKeys.PIPELINES_ACCESS_KEY` (= `"MODEL_MONITORING_PIPELINES_ACCESS_KEY"`) indeed doesn't exist in the writer pod
2. The suggestion returns `None`
3. `"V3IO_ACCESS_KEY"` is used as a fallback
4. The saving process works (at least for you)

I need to re-check which one of the two access keys is the right one to use here.